go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0
 	github.com/urfave/cli v1.22.5
 	go.opencensus.io v0.24.0
-	golang.org/x/net v0.8.0
+	golang.org/x/net v0.12.0
```

Comment:
Should I keep these or remove?

Reply:
why are you doing go get? oglest test is present by default in gcsfuse binary and should work to run unit tests.

go get command is used to fetch the latest version, because of which the dependencies version go updated.